.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish to PyPI
```

Comment:
can we make this step depending on successful run of linter and pytests?

Reply:
I couldn't find a way how, I mean there doesn't seem to be any kind of dependencies between gh actions. But test triggers only on a tag, and production only on a release, I can imagine that tag is being created after all the tests did pass